doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py

```diff
@@ -220,8 +220,15 @@ def go_to_pose_goal(self):
 
     move_group.set_pose_target(pose_goal)
 
-    ## Now, we call the planner to compute the plan and execute it.
-    plan = move_group.go(wait=True)
+    ## We call the planner to compute the plan and execute it.
+    plan = move_group.plan()
+    # "move_group.execute(plan, wait=True)" would execute the plan
```

Comment:
```suggestion
    # To execute the plan (if hardware is available):
    # "move_group.execute(plan, wait=True)" 
```

Reply:
But it would execute the motion in MoveIt's `demo.launch` simulation too.